003_Utils_L63/003_Utils_L63_safe.kt

```
@@ -0,0 +1,82 @@
+import java.awt.Desktop
+import java.io.File
+import java.math.BigInteger
+import java.net.URI
+import java.security.MessageDigest
+import java.util.*
+import java.net.URL
+
+fun hexStringToByteArray(strArg: String): ByteArray {
+    val HEX_CHARS = "0123456789ABCDEF"
+    val str = strArg.toUpperCase();
+
```

detekt (1.23.8)

[warning] This else block is empty and can be removed.

(detekt.empty-blocks.EmptyElseBlock)